tests/search/reducers/test_search.js

```diff
@@ -0,0 +1,8 @@
+import search from 'search/reducers/search';
+
+describe('search reducer', () => {
+  it('has a null query', () => {
+    const { query } = search(undefined, {type: '@@INIT'});
```

Comment:
Does the @@INIT action need to be explicitly passed here? 

It might be worth a comment or expand the first arg to `it()` with a bit more detail.